app/controllers/openid_connect/authorization_controller.rb

```diff
@@ -21,11 +21,20 @@ def index
       return redirect_to(sign_up_completed_url) if needs_sp_attribute_verification?
       link_identity_to_service_provider
       return redirect_to(user_authorization_confirmation_url) if auth_count == 1
-      handle_successful_handoff
+      aal3_requirement || handle_successful_handoff
```

Comment:
We're only allowing this for OIDC, not SAML, right?

Reply:
@zachmargolis yes, but I believe this should still be fine, as SAML will never mark AAL3 as required, so that requirement will always pass through to `handle_successful_handoff` for SAML.